quantum/quantum_keycodes.h

```diff
@@ -77,8 +71,6 @@ enum quantum_keycodes {
     QK_STENO_BOLT           = 0x5A30,
     QK_STENO_GEMINI         = 0x5A31,
     QK_STENO_MAX            = 0x5A3F,
-    QK_SWAP_HANDS           = 0x5B00,
-    QK_SWAP_HANDS_MAX       = 0x5BFF,
     QK_MOD_TAP              = 0x6000,
```

Comment:
Something that caused me problems was trying to use 0x5C00+ because it looks like it's available at this point in the file. Nothing makes figuring out what is going on harder than pressing MAGIC_* keycodes by accident.
```suggestion
    // Warning: 0x5C00+ are not free to use (see below)
    QK_MOD_TAP              = 0x6000,
```